app/assets/javascripts/discourse/app/components/sidebar/section.hbs

```diff
@@ -20,13 +20,18 @@
           <span class="sidebar-section-header-text">
             {{@headerLinkText}}
           </span>
+
           {{#if @indicatePublic}}
-            <span class="sidebar-section-header-global-indicator">
-              {{d-icon "globe"}}
-              <DTooltip @placement="top">{{d-icon "shield-alt"}}
-                {{i18n "sidebar.sections.global_section"}}
-              </DTooltip>
-            </span>
+            <DTooltip
+              @icon="globe"
+              class="sidebar-section-header-global-indicator"
+            >
+              <span
+                class="sidebar-section-header-global-indicator__content"
+              >{{d-icon "shield-alt"}}{{i18n
+                  "sidebar.sections.global_section"
+                }}</span>
```

Comment:
```suggestion
<span class="sidebar-section-header-global-indicator__content">
  {{d-icon "shield-alt"}}{{i18n "sidebar.sections.global_section"}}
</span>
```

Reply:
It's not going to render correctly if I do this, I would have to add more html elements